man/oci-runtime-tool-generate.1.md

```diff
@@ -60,6 +60,14 @@ read the configuration from `config.json`.
   The *fileMode*, *uid*, *gid* are optional.
     *fileMode* is the file mode of the device file.
     *uid*/*gid* is the user/group id of the device file.
+  This option can be specified multiple times.
+
+**--device-remove**=*TYPE:MAJOR:MINOR:PATH[:OPTIONS...]*
```

Comment:
I think it will be better to add, if only these optional fields are not the same, then it will delete the wrong device?